pyproject.toml

```diff
@@ -42,6 +42,9 @@ pytest-xdist = "^2.5.0"
 web3 = {extras=["tester"], version="6.0.0b4"}
 
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
If poetry changed pyproject.toml format, perhaps all dev dependencies could be moved to this category and re-locked?